src/parser_utils/dune

```diff
@@ -12,6 +12,8 @@
   flow_parser_utils_signature_builder
   flow_typing_polarity
   hh_json ; hack
+  flow_typing_type
+  flow_typing_errors
```

Comment:
Can you confirm that these are necessary?

Reply:
yes, otherwise I would get a compile error:                                      
"Error: Unbound module Type"